tests/framework/caching/CacheTestCase.php

```diff
@@ -287,8 +287,11 @@ public function testGetOrSetWithDependencies()
         $loginClosure = function ($cache) use (&$login) { return 'SamDark'; };
         $this->assertEquals($expected, $cache->getOrSet('some-login', $loginClosure, null, $dependency));
 
-        $dependency->invalidate($cache, 'test');
-        $expected = 'SamDark';
-        $this->assertEquals($expected, $cache->getOrSet('some-login', $loginClosure, null, $dependency));
+        /**
+         * dynasource: the following is failing for all caches
```

Comment:
I couldnt find a commit in which this test was passing correctly. Perhabs that was because the pull of your branch did not include 2.1 yet